### DISCUSSION TITLE
能不能支持下中兴B863av3.1-T2？

### DISCUSSION THREAD

**Discussion by stevenlinux** [ophub/amlogic-s9xxx-armbian#2022] [Amlogic]
![IMG20240114154954](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/3c57e60c-d5fd-409a-bd95-303855d3b34d)
![IMG20240114155014](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/6ad65643-65ba-4864-9c03-7fc8f28705f1)
![IMG20240114155027](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/7b886283-3ba0-4f02-9b39-63c18848c718)

**Comment by chouex** [comment]
是不是已经有了:https://www.znds.com/tv-1215785-1-1.html
注意是B863AV3.1-**M2**

**Reply by stevenlinux** [reply]
我有空刷下试试

**Reply by stevenlinux** [reply]
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/90007d4d-4151-4f39-bece-cd9614f7d3a5)
试了不行，m2的包刷不了T2

**Reply by chouex** [reply]
你是刷android的包? 不是要刷armbian嗎?
拆開散熱片看看CPU是不是s905L,
你試試把https://github.com/ophub/amlogic-s9xxx-armbian/releases/download/Armbian_jammy_save_2024.01/Armbian_24.2.0_amlogic_s905l_jammy_5.15.145_server_2024.01.01.img.gz 刷到USB, 先看看能不能usb boot, 有關usb boot有不同的方法

如果原系統打開了usb除錯, adb connect 盒子IP, adb shell reboot update, 也許會進入usb系統

進了系統先不要armbian-install

有條件可以看TTL輸出
我猜是
3.3v TX RX GND
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/767140/3bb3d391-8de7-4687-baf4-c4b04210e0b1)

**Reply by stevenlinux** [reply]
我看到的教程都是先刷android tv,在刷armbian。我试试能不能直接刷armbian

**Reply by stevenlinux** [reply]
电视盒子自带的运营商系统没有usb选项，自带系统应该没法从usb启动。![IMG20240116114453.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/965da4d6-b743-49cb-8e07-8236caf0b3b4)

![IMG_20240116_114553.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/90433bfc-3c55-46bb-bbc7-b6988c2aa297)

![IMG20240116114348.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/33886004/b2cf80dd-7dc5-4cab-8ad9-da9339745c99)

cpu上的散热片粘的很结实，不知道怎么拆掉。

**Reply by chouex** [reply]
建議你準備一條USB轉TTL線看下uboot, 可能可以直接進usb系統

開機時TTL出現輸出, 不停按回車, 直到出現shell
輸入`usb start; run recovery_from_udisk`

上面TTL看錯了, 我猜應該是
3.3v TX RX GND (RX TX可能反過來)
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/767140/e6cc0be0-2349-4114-9592-6e88af599bf6)
(3.3V不用接)
